packages/react/.storybook/main.cjs

```diff
@@ -1,3 +1,8 @@
+const viteTsconfig = require('vite-tsconfig-paths');
```

Comment:
If I understand well, it prevents to redeclare alias within the vite config? Am I right ?

Reply:
Exactly, the same way we do it on richie, but for webpack!

Reply:
Interesting ... for verna !